libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectLocationType.kt

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import androidx.annotation.IntDef
+
+/**
+ * Holds available [RoadObjectLocationType] types.
```

Comment:
Could you describe high level what is the location type?

Reply:
renamed